docs/develop/sdk/1-dev-setup.md

```diff
@@ -0,0 +1,150 @@
+---
+id: dev-setup
+title: Development Setup
+---
+import CodeBlock from '@theme/CodeBlock';
+import Example1 from '!!raw-loader!@site/code-examples/core_features/dev_setup.ts';
+
+If you want to develop a DApp, Wallet, web3 login or similar, you will need a running blockchain that doesn't require you to buy KILT just to develop and test the next big thing.
+For that you can use the peregrine network.
+The peregrine network is a parachain that is similar to the spiritnet (our mainnet) in functionality, but it's coin, the PILT, doesn't hold any monetary value.
+New features that we add to our blockchain, will first be tested on Peregrine before they are introduced to the Spiritnet.
+This gives you the chance to test your software with the newest blockchain features, before they are available on the mainnet.
+
+Even though the peregrine network is convenient, there are a scenarios where a public network is not the right choice for developers.
+If you need more funds than the faucet can provide, or if you need to reset the state of the blockchain, you will need to setup your own little KILT blockchain.
+
+Below, we will guide you through the process of connecting to the Peregrine network as well as running your own KILT blockchain.
+Moreover, we will explain the difference between our _Standalone_ and _Parachain_ runtimes Peregrine/Spiritnet.
+
+## Setup your Project
+
+Before we can connect to our blockchain, we need to create a small project that can connect to the chain.
```

Comment:
This is a bit confusing. I guess we are not revealing the dev accounts for the Peregrine network, right? Then we should make it clear somewhere.

Reply:
[4658905](https://github.com/KILTprotocol/docs/pull/81/commits/46589052755ad7ed8e637e6d8184913c46a4096b)

Reply:
hope that makes it a bit clearer?

Reply:
Good enough! 👍